docs/wiki/basic-tutorials/running.md

```diff
@@ -0,0 +1,315 @@
+
+![](./running-idea-configurations-create.png)
+
+**7.** Select your configuration in the drop-down menu and run or debug it with the buttons to the right.
+
+![](./running-idea-configurations-bar.png)
+
+### Eclipse
+
+This assumes your project is set up correctly, can be built without errors, and does not yet have any run/debug configurations.
+
+**1.** In the menu bar, click "Run" then "Run Configurations...".
+
+**2.** In the newly opened window, select "Java Application" on the left side, then click the leftmost button in the row above the tree view. A new configuration will appear.
+
+![](./running-eclipse-configurations-empty.png)
+
+**3.** Give a name to your configuration.
+
+**4.** Set the project and the main class. To easily select it, use the "Browse..." and "Search..." buttons.
+
+**5.** Optional: Set command line (and VM) arguments as well as environment variables in their respective tabs.
+
+**6.** Click `Apply` to save your configuration, then `Close` to close the window.
+
+![](./running-eclipse-configurations-create.png)
+
+**7.** Run or debug your bot via the Buttons in the top row, the `Run` menu, or the shortcuts Ctrl+F11 for running and F11 for debugging.
+
+![](./running-eclipse-configurations-bar.png)
+
+## :package: Deploying and Running as a Standalone Application
+
+Running from the IDE is only recommended during development and strongly discouraged for production use. Generally, you'll want your build tool to create a convenient distribution format for you to use.
+
+### Building a Distribution with Gradle
+<LatestVersion/>
+
+For Gradle, only two further steps are necessary for a basic application. On top of the steps described in the [Getting Started Section](/wiki/getting-started/intellij-gradle), also add the [Application Plugin](https://docs.gradle.org/current/userguide/application_plugin.html) and define your `mainClass` as the fully qualified name of your main class. If you're using an older version of Gradle (earlier than 6.4), the attribute is instead called `mainClassName`.
+
+::: tip INFO
+As with many Gradle solutions, there is actually a whole lot going on under the hood. The `application` plugin implicitly also applies the `java` and `distribution` plugins. Refer to the documentations of the involved plugins for more ways to fine-tune the process.
+:::
+
+Your modified build file should now look similar to this:
+
+<CodeGroup>
+  <CodeGroupItem title="build.gradle.kts" active>
+
+```kts
+plugins {
+    application
+}
+ 
+version = "1.0.0"
+ 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+}
+ 
+application {
+    mainClass.set("com.github.yourname.BotMain")
+    // mainClassName.set("com.github.yourname.BotMain")  // Gradle < 6.4
+}
+ 
+repositories {
+    mavenCentral()
+}
+ 
+dependencies {
+    implementation("org.javacord:javacord:{{latestVersion}}")
+}
+```
+
+  </CodeGroupItem>
+
+  <CodeGroupItem title="build.gradle">
+
+```gradle
```

Comment:
gradle does not have syntax highlighting here. Change back to `groovy`

Reply:
noticed this, did that.